src/Routes/Devices/Inventory.js

```diff
@@ -50,6 +50,40 @@ const Inventory = () => {
     setIsRowSelected(isRow);
   };
 
+  const setCanBeUpdated = () => {
+    let canBeUpdated = false;
+    if (checkedDeviceIds.length > 0) {
+      let initialImage = checkedDeviceIds[0].imageSetId;
+      for (let device of checkedDeviceIds) {
+        if (device.imageSetId !== initialImage) {
+          canBeUpdated = false;
+          break;
+        }
+        if (
+          (!canBeUpdated &&
+            device.updateImageData &&
+            device.deviceStatus === 'updateAvailable') ||
+          device.deviceStatus === 'error'
```

Comment:
@matoval I don't if that condition will cover all the cases. For example, if any devices with an error without an update available, it will return `canBeUpdated = true`
Does it make sense?

Reply:
That's a good point, I'm not sure if that is possible because could an error happen and the system still updates? Either way I can add a check that an update is available